### PR TITLE
[SDP-10] stub policies for existing models

### PR DESCRIPTION
**Pull request by cjarrett**

### CODE REVIEW

File: app/policies/event_policy.rb [hunk 1]
```diff
@@ -1,4 +1,24 @@
...
+  def create?
+    user.present?
+  end
+
+  def new?
+    user.present?
+  end
+
+  def update?
+    return true if user.present?
```
Comment (johndavid400) [inline app/policies/event_policy.rb]: Why are the create/new methods using user.present? and the update/destroy methods using return true if user.present?
Also, are they all supposed to be missing the 'edit' methods?

File: spec/factories/events.rb [hunk 2]
```diff
@@ -1,7 +1,7 @@
 FactoryBot.define do
   factory :event do
-    campaign_id 1
-    organization_id 1
+    association :campaign
```
Comment (johndavid400) [inline spec/factories/events.rb]: I like this syntax better, but Jacob and I discovered that they have a magic shortcut where you can just type the word campaign to setup the association... no value - just campaign. So he has used that syntax in some other tests.

Comment (cjarrett) [reply spec/factories/events.rb]: agreed, I tend to like the explicit syntax, but as long as we're keeping consistent all good. I'll change when I fix merge conflicts.

File: app/policies/event_policy.rb (hunk 1 above)
Comment (cjarrett) [reply app/policies/event_policy.rb]: I'll update these to be more relevant as the originals were stubs from Pundit.

**Comment by cjarrett** [comment]
Policies feel a little better now. We can flesh them out more once we start building out features.

**Comment by cjarrett** [comment]
Ready for review

**Pull request merged**